### DISCUSSION TITLE
请问安卓端是否支持从自定义url下载geodata数据

### DISCUSSION THREAD

**Discussion by CC-3301** [MetaCubeX/ClashMetaForAndroid#207] [Q&A]
开发组您好，我自己维护了一个geodata数据库，里面有我新增的geosite:anti-ad数据集，在linux端使用clash meta时可以正常加载，但在安卓端就提示报错，使用自定义数据集必须手动导入吗？

配置文件
geox-url:
  geoip: "https://github.com/ShoukakuChuuCC/geodata/releases/download/latest/geoip.dat"
  geosite: "https://github.com/ShoukakuChuuCC/geodata/releases/download/latest/geosite.dat"

Linux端正常加载
![image](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/38343280/998825c9-216e-4096-bfbe-0fceceb3c7f7)

安卓端报错
![image](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/38343280/d44a022f-01d7-4b03-a017-3f2c90b2a350)

**Comment by lim-kim930** [comment]
我去，我刚准备问呢，互联网嘴替啊

**Comment by CC-3301** [comment]
我要分享自己写的配置订阅给朋友使用，如果手动导入的话稍显麻烦

**Reply by lim-kim930** [reply]
绝了，一模一样的需求的😂

**Comment by PingZi-Wing** [comment]
我手动导入geoip库也不能触发IP，电报不通

**Comment by Kwisma** [comment]
这问题还没修复吗？我很需要这个功能啊。
手动导入太麻烦了，需要自动导入的才好

**Comment by GreyFlorence** [comment]
遇到同样的困扰，目前Geosite只能随着APP更新而更新，对新规则的支持不够及时。希望能解决

**Comment by mengdegege** [comment]
建议放弃geosite，改用rule-set

**Comment by eronez** [comment]
https://github.com/eronez/ClashMetaForAndroid/releases/tag/cmfa-2.11.22-alpha-with-geo

Try this, but not fully tested.
Works for me.
